build.sbt

```diff
@@ -29,6 +29,10 @@ addCommandAlias(
   "publishSnapshots",
   s"; ++ ${SCALA_2_12}; projectJVM2_13/publish; projectJVM2_12/publish; projectJS/publish; sbtAirframe/publish;"
 )
+addCommandAlias(
+  "publishJVMLocal",
+  s"; ++ ${SCALA_2_12}; projectJVM2_13/publishLocal; projectJVM2_12/publishLocal; projectJS/publishLocal; sbtAirframe/publishLocal;"
```

Comment:
publishJSLocal can cover `projectJS/publishLocal`, so looks like projectJS can be removed from publishJVMLocal.

Reply:
nice catch. Thanks